content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -37,12 +37,11 @@ further_reading:
 
 ## Compatibility requirements
 
-The .NET Tracer supports instrumentation on:
-  - .NET 5
-  - .NET Core 3.1
-  - .NET Core 2.1
+### Supported .NET Core runtimes
 
-For a full list of supported libraries, visit the [Compatibility Requirements][1] page.
+The .NET Tracer supports instrumentation on .NET Core 2.1 and above.
```

Comment:
I would keep the list we had with the 3 specific versions. ".NET Core 2.1 and above" includes 2.2 and 3.0, which we do not support.

Reply:
Updated - since the builds are working here is a screenshot: https://a.cl.ly/eDuymDj9

Reply:
Added here https://github.com/DataDog/documentation/pull/11200/commits/9adac379e411099aa53151c32c4a9f5f9e5c7271